src/file_watching.jl

```diff
@@ -24,7 +24,10 @@ Check if a `WatchedFile` has changed. Returns -1 if the file does not exist, 0 i
 has not changed, and 1 if it has changed.
 """
 function has_changed(wf::WatchedFile)
-    isfile(wf.path) || return -1
+    if !isfile(wf.path)
+      sleep(0.1)
+      isfile(wf.path) || return -1
```

Comment:
```suggestion
        sleep(0.1)
        isfile(wf.path) || return -1
```
Indentation appears to not be 4 spaces.